andela/ah-titans-frontend
# #160088425 Implement designs signup/registration

#### What does this PR do?
Design an implementation for signup container.

#### Description of tasks to be completed
Implement designs for the Signup/Registration Page.

#### How should this be manually tested?
```sh
$ git clone https://github.com/andela/ah-titans-frontend.git
```
```sh
$ git checkout ft-implement-designs-signup/registration-160088425
```
```sh
$ cd ah-titans-frontend
```

>Install the dependencies using the command below.

```sh
$ npm install
```
>Run the application as below.
```sh
$ npm start
```
>Now you can navigate to the link 'https://http://localhost:8080/signup' on your browser and you should be able to see a page as the one shown on the screenshot below.

<img width="1280" alt="screen shot 2018-08-30 at 15 13 38" src="https://user-images.githubusercontent.com/27728464/44853591-06c94080-ac6f-11e8-98be-e380fbcf08c9.png">


#### Any background context you want to provide?
No

#### What are the relevant Pivotal Tracker Stories?
[#160088425](https://www.pivotaltracker.com/story/show/160088425)
